client/src/app/_guards/edit.guard.ts

```diff
@@ -19,9 +18,7 @@ export class EditGuard implements CanActivate {
             if ((connectedUser.specialAccount && connectedUser.specialAccount.id === id)
                 || (connectedUser.barman && connectedUser.barman.id === id)
                 || this.ngxPermissionsService.getPermissions()['barman:write']
-                || this.ngxPermissionsService.getPermissions()['specialaccount:write']) {
-                return true;
-            }
+                || this.ngxPermissionsService.getPermissions()['specialaccount:write']) return true;
```

Comment:
I'm still not convinced about the _no braces if one line_ ...
We should not restrict someone to make somthing more readable